### ISSUE TITLE
Is there a way to do this in a client-side template?

### ISSUE THREAD

**Issue by harmtemolder** [gtm-templates-simo-ahava/sha256-hasher#1]
The `sha256Sync` API is not available client-side, and I tried using the [`sha256` API](https://developers.google.com/tag-platform/tag-manager/templates/api#sha256), but I couldn't get that to `return` the hashed input. Is there a way to do this in a client-side template?

**Comment by wunaidage** [comment]
I dont think you can use sha256 API on GTM. My approach is to do hash in my project code then add to data layer
